tests/features/S2N_LIBCRYPTO_SUPPORTS_EVP_KEM.c

```diff
@@ -24,7 +24,8 @@ int main()
     }
     if (!EVP_PKEY_CTX_kem_set_params(ctx, NID_KYBER512_R3)
             || !EVP_PKEY_CTX_kem_set_params(ctx, NID_KYBER768_R3)
-            || !EVP_PKEY_CTX_kem_set_params(ctx, NID_KYBER1024_R3)) {
+            || !EVP_PKEY_CTX_kem_set_params(ctx, NID_KYBER1024_R3)
+            || !EVP_PKEY_CTX_kem_set_params(ctx, NID_MLKEM768)) {
```

Comment:
This will cause a regression for customers that aren't using the latest AWS-LC. We will likely need a separate probe for MLKEM768 to avoid that.

Reply:
I suspect that will result in a lot of branching logic to deal with the different versions... Just so we're clear, you think that rather than renaming this file to `S2N_LIBCRYPTO_SUPPORTS_EVP_KEM.c` we should keep the old file named for Kyber and create a new one that has the ML-KEM NID check?

Reply:
> I suspect that will result in a lot of branching logic to deal with the different versions

Is there really an alternative, though? If an application pulls in a new version of s2n-tls and doesn't also upgrade aws-lc it will result in disabling PQ capabilities entirely. That is a problem.